tests/push.bats

```diff
@@ -101,3 +101,46 @@ load helpers
   run_buildah 1 push --signature-policy ${TESTSDIR}/policy.json busybox docker://registry.example.com/evenbusierbox
   expect_output --substring 'push to registry at "registry.example.com" denied by policy: not in allowed registries list'
 }
+
+
+@test "buildah push image to containers-storage" {
+  buildah pull --signature-policy ${TESTSDIR}/policy.json busybox
+  run_buildah push --signature-policy ${TESTSDIR}/policy.json busybox containers-storage:newimage:latest
+  run_buildah images
+  expect_output --substring "newimage"
+  buildah rmi newimage busybox
+}
+
+@test "buildah push image to docker-archive and oci-archive" {
+  buildah pull --signature-policy ${TESTSDIR}/policy.json busybox
+  for dest in docker-archive oci-archive; do
+    mkdir ${TESTDIR}/tmp
+    run_buildah push --signature-policy ${TESTSDIR}/policy.json busybox $dest:${TESTDIR}/tmp/busybox.tar:latest
+    run ls ${TESTDIR}/tmp/busybox.tar
+    [[ $status -eq 0 ]]
+    rm -rf ${TESTDIR}/tmp
+  done
+  buildah rmi busybox
+}
+
+@test "buildah push image to docker and docker registry" {
+  run which docker
+  if [[ $status -ne 0 ]]; then
+    skip "docker is not installed"
+  fi
+
+  buildah pull --signature-policy ${TESTSDIR}/policy.json busybox
+  run_buildah push --signature-policy ${TESTSDIR}/policy.json busybox docker-daemon:buildah/busybox:latest
+  run docker images
```

Comment:
ditto run

Reply:
This run is used for update the $output context for the following expect_output check. So will keep it.